docs/features/writing-different-telemetry-types.md

```diff
@@ -42,6 +45,59 @@ We provide support for the following dependencies:
 
 Since measuring dependencies can add some noise in your code, we've introduced `DependencyMeasurement` to make it simpler. ([docs](#making-it-easier-to-measure-dependencies))
 
+### Measuring Azure Blob Storage dependencies
+
+We allow you to measure Azure Blob Storage dependencies.
+
+Here is how you can report a dependency call:
+
+```csharp
+var durationMeasurement = new Stopwatch();
+
+// Start measuring
+durationMeasurement.Start();
+var startTime = DateTimeOffset.UtcNow;
+
+_logger.LogBlobStorageDependency(accountName: "multimedia", containerName: "images", isSuccessful: true, startTime, durationMeasurement.Elapsed);
+// Output: "Dependency Azure blob multimedia named images in 00:00:00.2521801 at 03/23/2020 09:56:31 +00:00 (Successful: True - Context: )"
+```
+
+### Measuring Azure Cosmos dependencies
+
+We allow you to measure Azure Cosmos dependencies.
+
+Here is how you can report a dependency call:
+
+** Cosmos SQL**
+
+```csharp
+var durationMeasurement = new Stopwatch();
+
+// Start measuring
+durationMeasurement.Start();
+var startTime = DateTimeOffset.UtcNow;
+
+_logger.LogCosmosSqlDependency(accountName: "administration", database: "docs", container: "purchases", isSuccessful: true, startTime: startTime, duration: durationMeasurement.Elapsed);
+// Output: "Dependency Azure DocumentDB docs/purchases named administration in 00:00:00.2521801 at 03/23/2020 09:56:31 +00:00 (Successful: True - Context: )"
```

Comment:
Is it correct that here the dependency target name is the account name, while in the blob storage tracking the account name is the dependency data?

Reply:
That's a good question 🤔 Let's turn it around for storage as well so that it's the account being the target

Reply:
Ok in seperate PR

Reply:
Moved to #106